base/sparse/sparsevector.jl

```diff
@@ -144,10 +167,17 @@ sparsevec(I::AbstractVector, v::Number, len::Integer, combine::Function) =
 
 ### Construction from dictionary
 """
-    sparsevec(D::Dict, [m])
+    sparsevec(d::Dict, [m])
 
 Create a sparse vector of length `m` where the nonzero indices are keys from
 the dictionary, and the nonzero values are the values from the dictionary.
+
+```jldoctest
+julia> sparsevec(Dict(1 => 3, 2 => 2, 2 => 4))
```

Comment:
for a minute I thought "oh that's interesting, sparsevec from a Dict doesn't do combining" but no, the overwriting happens in the Dict constructor so the duplicate key is probably confusing to have here